### DISCUSSION TITLE
Compressing libraries with zip: true not working for me, is it for anyone else?

### DISCUSSION THREAD

**Discussion by bjornhansen** [UnitedIncome/serverless-python-requirements#839] [Q&A]
Under the "Dealing with Lambda's size limitations" section of the readme, it suggests compressing libraries to save space. But when I try to use it as it is in the docs, I get import errors.

I'm using

```
custom:
  pythonRequirements:
    zip: true
```

in `serverless.yml` and

```
try:
  import unzip_requirements
except ImportError:
  pass
```

in my handler module at the very top.

Any ideas why this isn't working? Has anyone else got it working? I'm using FastAPI with mangum ASGI adapter, would this throw off the import paths?

**Comment by cvargas-xbrein** [comment]
mm try to add too, in my case works fine, try to reinstall sls or node
zipImport: true
